### PR TITLE
Split ABC definitions into a standalone repo

### PR DESCRIPTION
**Pull request by elprans**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -164,9 +164,9 @@ def run(self):
 
 
 setup(
-    name='azure-functions-python',
+    name='azure-functions-worker',
```
Comment (pragnagopa) [inline setup.py]: rename to match the package azure.functions_worker

File: setup.py [hunk 2]
```diff
@@ -164,9 +164,9 @@ def run(self):
...
     version='1.0.0a2.dev0',
-    description='Python Support for Azure Functions',
+    description='Python Worker for Azure Functions Runtime',
```
Comment (pragnagopa) [inline setup.py]: Nit: Python Language Worker for Azure Functions Host

File: setup.py [hunk 3]
```diff
@@ -178,18 +178,17 @@ def run(self):
...
+    packages=['azure.functions_worker',
               'azure.functions_worker.protos',
               'azure.functions_worker.bindings'],
-    provides=['azure.functions'],
     setup_requires=[
         'grpcio~=1.9.1',
         'grpcio-tools~=1.9.1',
     ],
     install_requires=[
         'grpcio~=1.9.1',
```
Comment (pragnagopa) [inline setup.py]: update to latest grpc 1.12.0

File: setup.py (hunk 1 above)
Comment (elprans) [reply setup.py]: Python package names customarily have dots and underscores replaced by dashes.  See, for example, https://pypi.org/project/azure-mgmt-storage/

**Pull request merged**